sdk/identity/identity/src/credentials/interactiveBrowserCredential.browser.ts

```diff
@@ -1,17 +1,21 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import * as msal from "msal";
```

Comment:
I see `msal` in the dependencies section in the package.json. Do we use it elsewhere or we can remove it from the dependency list?

Reply:
In this new approach we're using both MSAL 1 and MSAL 2, so we need to keep this package. We're keeping Implicit Grant flow as an opt-in feature. If users have issues with our new default (Auth Code flow with PKCE), they can switch to the previous one by sending the `flow: "implicit-grant"` parameter on the InteractiveBrowserCredential constructor.